### ISSUE TITLE
Currency and fund codes

### ISSUE THREAD

**Issue by fjelsted** [mledoze/countries#290]
Currency and fund codes are bundled together making the currency information less useful regarding certain countries, e.g. Switzerland, USA and Chile.

May I suggest getting rid of fund codes? They have limited use outside of accounting. Either get rid of them or separate them out to a different field.

On the [XML list](https://www.currency-iso.org/en/home/tables/table-a1.html) funds are attributed with `isFund="true"`.

Other countries still have multiple currencies that are actually in circulation, e.g. Cuba, so currencies still have to be kept in array form.

**Comment by mledoze** [comment]
Hello @fjelsted what do you mean by "fund codes". As far as I know, there only currency codes in the dataset.

**Comment by fjelsted** [comment]
Hi @mledoze. :) If you take a look at USA for instance:
```
{
  "name": {
    "common": "United States",
    "official": "United States of America",
    "native": {
      "eng": {
        "official": "United States of America",
        "common": "United States"
      }
    }
  },
  "tld": [".us"],
  "cca2": "US",
  "ccn3": "840",
  "cca3": "USA",
  "cioc": "USA",
  "independent": true,
  "status": "officially-assigned",
  "currency": ["USD", "USN", "USS"],
  [...]
}
```
It has three currencies listed, which are all technically correct, but USS and USN are not currencies in the normal sense when we think "currencies". From my limited understanding of it, _funds_ are a group of currencies which serve as tools for the financial sector. For instance, MXV is a currency that is denoted in MXN (Mexican pesos) and adjusted by the Central Bank of Mexico to be unaffected by inflation. Apparently it's being used to denominate mortgage loans.

On https://www.currency-iso.org/ their _Current currency & funds code list_ (XML) marks funds with an `IsFund` attribute which should make it easy to seggregate. Taking USA as an example again:
```
<CcyNtry>
<CtryNm>UNITED STATES OF AMERICA (THE)</CtryNm>
<CcyNm>US Dollar</CcyNm>
<Ccy>USD</Ccy>
<CcyNbr>840</CcyNbr>
<CcyMnrUnts>2</CcyMnrUnts>
</CcyNtry>
<CcyNtry>
<CtryNm>UNITED STATES OF AMERICA (THE)</CtryNm>
<CcyNm IsFund="true">US Dollar (Next day)</CcyNm>  <------ `IsFund` attribute
<Ccy>USN</Ccy>
<CcyNbr>997</CcyNbr>
<CcyMnrUnts>2</CcyMnrUnts>
</CcyNtry>
```

**Comment by fjelsted** [comment]
I've never made a pull request before but the change seems pretty straight forward. I'll look into it. 👍

**Comment by mledoze** [comment]
Fixed by #293.
